groups/groups.yaml

```diff
@@ -1007,7 +1007,6 @@ groups:
       - dmaceachern@vmware.com
       - hhorl@pivotal.io
       - idealhack@gmail.com
-      - kubernetes-release-managers-private@googlegroups.com
```

Comment:
Is this needed? I think including an unmanaged group weakens the value of managing this group with a manifest.

Reply:
Unsure. I think this was @justaugustus.